### DISCUSSION TITLE
additional subdirectories added to my github/git structure

### DISCUSSION THREAD

**Discussion by pfarrell51** [community/community#46077] [New to GitHub]
While I am not literally new to github/git, I only know how to do a few fundamental things and
it has served me well.

Recently I have been using the github website to look at my repositories and have noticed
a change in the file folder structure that I didn't expect.

For a long time, I have had a repository/subdirectory called "gows"
that I use to keep projects I do in the 'go' language.

For a long time, it was simply
https://github.com/pfarrell51/gows

with separate folders for each project/executable

https://github.com/pfarrell51/gows/musictools

But today, I see two new subfolders: "tree/master"
between the normal 'gows' repository and the projects. 

https://github.com/pfarrell51/gows/tree/master/musictools

I have no idea where or when "tree/master" was inserted.
It may have been that I mistyped a "git push origin master" command
but I don't  know.

Can anyone tell me what these are? or how they got there?
And more importantly, how do I get ride of them so the
musictools project will just be in 

https://github.com/pfarrell51/gows/musictools

Thanks
Pat

**Comment by chris-c-thomas** [comment]
This is normal. While `tree/master` are not physically directories in your repository they will show up in this manner to let you know the branch path, etc of the files you're currently working on. Let's say you have 5 branches and you wanted to access `/pfarrell51/gows/musictools/music.go` ... how would you find and differentiate that from a URL perspective for your master branch, your develop branch, your musicFeatures branch, etc, etc. They'd all have the same URL. 

This is being inserted to follow the tree path of your current branch. It won't be in your local file structure and only visible on GitHub.com

**Reply by pfarrell51** [reply]
OK, thanks. That explains a bit of it. But I don't have any branches. Or have not yet needed any branches or created any branches. Where did the "tree/master" come about? Did Github just automagically create it?
